samples/routing/fair-distribution/Core_6/Client/Program.cs

```diff
@@ -83,9 +83,9 @@ static void AddRouting(EndpointConfiguration endpointConfiguration)
         #region Routing
 
         const string server = "Samples.FairDistribution.Server";
-        var routing = endpointConfiguration.Routing();
+        var routing = endpointConfiguration.UseTransport<MsmqTransport>().Routing();
         routing.RouteToEndpoint(typeof(PlaceOrder), server);
-        routing.Mapping.Physical.Add(
+        routing.GetSettings().GetOrCreate<EndpointInstances>().Add(
```

Comment:
@SimonCropp we wanted to avoid users hardcoding routing information. That API can be used by plugins to provide routing information from a custom information source and in that case you will most probably already have a feature or it's ok to expect the users plugin this functionality with a feature.